packages/firestore/src/api/database.ts

```diff
@@ -398,14 +398,30 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
           'any other methods on a Firestore object.'
       );
     }
+
+    let synchronizeTabs = false;
+
+    if (settings) {
+      if (settings.experimentalTabSynchronization !== undefined) {
+        log.error(
+          'The `experimentalTabSynchronization` setting has been renamed to ' +
+            '`synchronizeTabs`. In a future release, the setting  will be removed ' +
```

Comment:
extra space in `setting  will`

Reply:
Ugh, I thought I fixed this before sending this out. It's fixed now.